source/backend/api/Services/DocumentQueueService.cs

```diff
@@ -217,14 +217,6 @@ public async Task<PimsDocumentQueue> Upload(PimsDocumentQueue documentQueue)
             }
             databaseDocumentQueue.DocProcessStartDt = DateTime.UtcNow;
 
-            // if the document queued for upload is already in an error state, update the retries.
-            if (databaseDocumentQueue.DocumentQueueStatusTypeCode == DocumentQueueStatusTypes.PIMS_ERROR.ToString() || databaseDocumentQueue.DocumentQueueStatusTypeCode == DocumentQueueStatusTypes.MAYAN_ERROR.ToString())
-            {
-                this.Logger.LogDebug("Document Queue {documentQueueId}, previously errored, retrying", documentQueue.DocumentQueueId);
-                databaseDocumentQueue.DocProcessRetries += 1;
-                databaseDocumentQueue.DocProcessEndDt = null;
-            }
-
             bool isValid = ValidateQueuedDocument(databaseDocumentQueue, documentQueue);
```

Comment:
the above logic needs to run after this check - to avoid the processDocRetries from being changed.